CMakeLists.txt

```diff
@@ -394,9 +394,9 @@ if(UNITTEST)
              set_tests_properties(test_CML_ldpcut PROPERTIES PASS_REGULAR_EXPRESSION "Nerr: 0")
 
     # check channel simulator measures correct Peak to Average Power Ratio (about 0dB) with a sine wave input signal
-    add_test(NAME test_cohpsk_ch_papr
+    add_test(NAME test_ch_papr
              COMMAND sh -c "cd ${CMAKE_CURRENT_BINARY_DIR};
-                            ./misc/mksine - 1000 10 | ./src/cohpsk_ch - /dev/null -100 --Fs 8000 --ctest")
+                            ./misc/mksine - 1000 10 | ./src/ch - /dev/null --ctest")
```

Comment:
Should this be `$<TARGET_FILE:ch>` as well?

Reply:
As well as the equivalent for mksine, if built as part of the build process.

Reply:
The `cd ${CMAKE_CURRENT_BINARY_DIR};` puts us in the correct directory so the following lines work.  I've adopted this style as the same code can be pasted to the cmd line and run manually while developing/debugging tests.  The `$<TARGET_FILE:ch>` style only works inside cmake (you can't past the line to the cmd line for manual development) and is less intuitive for people not very familiar with cmake.

This and the other tests seem to run OK.

Reply:
Makes sense. For some reason I thought CMake applied its own interpolation before sh.